textalyze.js

```diff
@@ -17,7 +21,50 @@ function itemCounts(array) {
   // Your code here.
   // Run 'npm test' to see what tests need to pass.
 
+  array.forEach((value) => {
+    const sum = counts.get(value) ? counts.get(value) + 1 : 1;
+    counts.set(value, sum);
+  });
+
   return counts;
 }
 
-module.exports = { itemCounts };
+/**
+ * Takes an arbitrary String as input and returns an Array of all the characters in the string
+ * @param {String}
+ * @returns {Array}
+ */
+function stringCharsToArray(string = '') {
+  const arrayString = string.split('');
+  return itemCounts(arrayString);
+}
+
+/**
+ * Takes an arbitrary String as input and returns a "sanitized" string
+ * that replaces all upper-case letters with their lower-case equivalent
+ * @param {String}
+ * @returns {Array}
+ */
+function sanitize(string = '') {
+  return string.toLowerCase();
+}
+
+/**
+ * Takes an string containing the path to a file and returns the data
+ * @param {String} path to the file
+ * @returns {String} file data
+ */
+function readFromFile(file) {
+  return new Promise((resolve, reject) => {
+    fs.readFile(file, (err, data) => {
+      if (err) {
+        return reject(new Error(err));
```

Comment:
Isn't `err` already an `Error` object?  Why not just `return reject(err)`?

Reply:
Somehow jest couldn't identify the `throw` error that way